sdk/python/tests/integration/feature_repos/universal/online_store/hbase.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_for_logs
+
+from tests.integration.feature_repos.universal.online_store_creator import (
+    OnlineStoreCreator,
+)
+
+
+class HbaseOnlineStoreCreator(OnlineStoreCreator):
```

Comment:
Could you add a `Hbase` to the list of services that have containerized tests in `CONTRIBUTING.md`?

Reply:
Done!